draft-ietf-lamps-csr-attestation.md

```diff
@@ -679,6 +680,33 @@ network protocol, then the interaction would conceptually be:
 ~~~
 {: #fig-csr-client-p11 title="Example interaction between CSR generator and HSM."}
 
+## Separation of RA and CA roles with respect to Attestation Results {#sec-impl-ar}
+
+As described in {{architecture}}, CSRs MAY contain either Evidence or Attestation Results (AR),
+and also the Registration Authority (RA) and Certification Authority (CA) MAY be conceptualized as
+a single Relying Party entity, or as separate entities. There are some implications here worth discussion.
+
+In many cases, the Evidence contained within a CSR is intended to be consumed by the RA and not
+to be placed into the issued certificate.
```

Comment:
Are there cases where Evidence does go into the resulting certificate?
(I didn't think so)

Reply:
I do not think so either. This would be a bit too verbose IMHO and could cause all sorts of privacy challenges.